static/app/views/settings/organizationDeveloperSettings/sentryFunctionsEnvironmentVariables.tsx

```diff
@@ -0,0 +1,136 @@
+import styled from '@emotion/styled';
+
+import Button from 'sentry/components/button';
+import {InputField} from 'sentry/components/forms';
+import {PanelBody, PanelHeader} from 'sentry/components/panels';
+import {IconAdd, IconDelete} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import space from 'sentry/styles/space';
+
+type Props = {
+  envVariables: {
+    name?: string | undefined;
```

Comment:
@vpandiarajan20 you don’t need to make the field optional and also possibly undefined